torch/distributed/distributed_c10d.py

```diff
@@ -400,6 +401,15 @@ def pg_backend_config(self) -> Dict[ProcessGroup, str]:
         global _pg_backend_config
         return _pg_backend_config
 
+    @property
+    def backend_device(self) -> Dict[str, List[str]]:
+        """
+        Process backend's device
+        TODO don't expose the map, expose fine grained ops
```

Comment:
Should this be private for now if we know we want to change this API?

Reply:
modified